kview/src/main/java/dev/ikm/komet/kview/controls/skin/FilterOptionsPopupSkin.java

```diff
@@ -390,7 +385,7 @@ private void setAvailableOptionsFromNavigator(FilterOptions options, Navigator n
         if (control.getFilterType() == FilterOptionsPopup.FILTER_TYPE.NAVIGATOR) {
             // header: All first children of root
             List<String> headerList = navigator.getChildEdges(rootNid).stream()
-                    .map(edge -> navigator.getViewCalculator().getPreferredDescriptionTextOrNid(edge.destinationNid()))
+                    .map(edge -> FilterOptionsUtils.getPreferredDescriptionTextOrNid(navigator.getViewCalculator(), edge.destinationNid()))
```

Comment:
Can we change this to `.getDescriptionTextOrNid(nid)`?
reason: get preferred has an issue with always choosing regular name.

Reply:
Done!